x-pack/platform/plugins/shared/cases/server/common/types/case.ts

```diff
@@ -50,6 +50,7 @@ export interface CasePersistedAttributes {
   category?: string | null;
   customFields?: CasePersistedCustomFields;
   observables?: Observable[];
+  incremental_id?: number;
```

Comment:
```suggestion
  incremental_id?: number | null;
```

Reply:
It's actually never `null`, only `undefined`.